deploy/awstools/awstools.py

```diff
@@ -472,6 +476,7 @@ def main(args):
     parser.add_argument("--block_devices", type=yaml.safe_load, default=run_block_device_dict(), help="List of dicts with block device information. Used by \'launch\'.")
     parser.add_argument("--tags", type=yaml.safe_load, default=run_tag_dict(), help="Dict of tags to add to instances. Used by \'launch\'.")
     parser.add_argument("--filters", type=yaml.safe_load, default=run_filters_list_dict(), help="List of dicts used to filter instances. Used by \'terminate\'.")
+    parser.add_argument("--ami", default=get_f1_ami_id(), help="AMI ID to use. Defaults to F1 AMI ID. Used by \'launch\'.")
```

Comment:
```suggestion
    parser.add_argument("--ami", default=get_f1_ami_id(), help="AMI ID to use for 'launch'. Defaults to awstools.get_f1_ami_id().")
```

When I read "F1 AMI ID", I find myself asking "which F1 AMI"?  Using the function name makes it clearer what you mean and gives someone a pointer of where to look in the code.  If you dislike having the function name in the helpstring, you  could say `f"Defaults to {f1_ami_name}"`